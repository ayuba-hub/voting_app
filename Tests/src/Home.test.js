import {shallow} from 'enzyme';
import React from 'react';
import Home from './Home';

describe("Test On Home.js", ()=> {
    
    it('expects to render Home component', () => {
        expect(shallow(<Home/>).length).toEqual(1)
    })
    
    
    it('expects to render Home component', () => {
        expect(shallow(<Home/>)).toMatchSnapshot();
    })
    
})


