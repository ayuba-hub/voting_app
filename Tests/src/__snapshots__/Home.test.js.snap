// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test On Home.js expects to render Home component 1`] = `ShallowWrapper {}`;
